ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -57,8 +59,18 @@ class JITModBuilder {
   ValueToValueMapTy RevVMap;
 #endif
 
+  Value *getMappedValue(Value *V) {
```

Comment:
Silly question perhaps: *which* Map? I'd have imagined there can be more than one, at least realistically in the future?

Reply:
I see what you are saying, but the LLVM terminology is (for better or probably worse) `ValueMapper`, as in it maps `Value *` to `Value *` :P

> #include <llvm/Transforms/Utils/ValueMapper.h>

Reply:
Yes, but we can use a value mapper to do different things I assume? Is there a way of giving *this* valuemapper a more descriptive name?

Reply:
Well, we have a second map for debug builds, which we called the "reverse vmap" as opposed to the (forward) vmap.

As far as I can see, we won't need more, but let's see what @ptersilie thinks.

Reply:
Even so, if we can give it a better name, we should.

Reply:
I think we only need multiple maps if we want to inline recursive calls. At least that was one case where I experimented with multiple maps (which frankly was a nightmare so I hope I won't have to do that again).

So for now there's only one map. If you want to future proof this, we would have to pass the map to use into the `getMappedValue` function, which for now would serve no purpose.